share/vss/vss.go

```diff
@@ -0,0 +1,579 @@
+// vss.go implements the verifiable secret sharing scheme by Pedersen recalled
+// in "Provably Secure Distributed Schnorr Signatures and a (t, n) Threshold
+// Scheme for Implicit Certificates"  by Stinson and Strobl.
+// The basic scheme is taken from Pedersen's paper: "Non-interactive and
+// information-theoretic secure verifiable secret sharing", in Crypto ’91, pages
+// 129–140, 1991.
```

Comment:
@nikkolasg Not sure if it's good to mention two papers in the preamble here, it might be confusing to the reader. I think it's enough to reference the paper of Stinson and Strobl, especially since we already mentioned that it's a Pedersen scheme above.